core/scripts/vrfv2plus/testnet/proofs.go

```diff
@@ -97,21 +99,29 @@ func generateProofForV2Plus(e helpers.Environment) {
 	if err != nil {
 		helpers.PanicErr(fmt.Errorf("unable to parse preseed: %w", err))
 	}
-	preSeedData := proof.PreSeedDataV2{
+
+	parsedSubId, ok := new(big.Int).SetString(*subId, 10)
+	if !ok {
+		helpers.PanicErr(fmt.Errorf("unable to parse subID: %s %w", subId, err))
+	}
+	extraArgs, err := extraargs.ExtraArgsV1(*nativePayment)
+	helpers.PanicErr(err)
+	preSeedData := proof.PreSeedDataV2Plus{
 		PreSeed:          preSeed,
```

Comment:
Ah I see we provide the preseed to the script. This is fine but the preseed onchain should be different now that sub id is uint256.

Reply:
yes the preseed onchain should be different now. I believe this was already handled in https://github.com/smartcontractkit/chainlink/pull/9884 as proofs for foundry tests were regenerated after global subID changes